storybook/cypress/e2e/date-picker.cy.ts

```diff
@@ -10,10 +10,11 @@ describe("Date Picker", () => {
       .and("not.be.empty")
       .then((id) => {
         cy.get(`label[for="${id}"]`).should("exist").and("not.be.empty");
-      });
 
-    cy.checkA11y("#root-inner");
-    // cy.percySnapshot();
+        cy.checkA11y("#root-inner");
+
+        cy.get("#root-inner").matchImageSnapshot();
```

Comment:
We gingen toch geen `#root-inner` meer doen?

Voor `cy.checkA11y()` is er een andere reden, maar kunnen we hier ook nog iets mee?